src/MLJCatBoostInterface.jl

```diff
@@ -28,6 +28,40 @@ function mlj_to_kwargs(model)
                             for name in fieldnames(typeof(model)))
 end
 
+"""
+    get_dtype_feature_ix(X, dtype)
+
+Get the index of the columns with a specific dtype
+
+Parameters
+----------
+- `X`: Table
+- `dtype`: DataType 
+
+Returns
+-------
+- `Vector{Int64}`
+"""
+function get_dtype_feature_ix(X, dtype)
+    return findall(eltype.(MMI.scitype.(eachcol(X))) .<: dtype)
```

Comment:
Perhaps simpler is `return findall(MMI.schema(X).scitypes .< dtype)`

Reply:
Yes, that is simpler. Thank you! I have added it in the latest commit.